pkg/controller/eventhandlers/eventhandlers.go

```diff
@@ -31,7 +31,7 @@ import (
 // EventHandler accepts a workqueue and returns ResourceEventHandlerFuncs that enqueue messages to it
```

Comment:
Can you change `ResourceEventHandlerFuncs` in the comment?

Reply:
Done